packages/framework/attributor/api-report/attributor.api.md

```diff
@@ -63,7 +63,7 @@ export const mixinAttributor: (Base?: typeof ContainerRuntime) => typeof Contain
 
 // @internal
 export class OpStreamAttributor extends Attributor implements IAttributor {
-    constructor(deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>, audience: IAudience, initialEntries?: Iterable<[number, AttributionInfo]>);
+    constructor(deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>, quorumClients: IQuorumClients, initialEntries?: Iterable<[number, AttributionInfo]>);
```

Comment:
I think `OpStreamAttributor` and `Attributor` were exported from this package for reasons that no longer apply. Could you just remove the export from `index.ts` and the two internal release tags as part of this PR?